language_service/src/generate_docs/tests.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#![allow(clippy::needless_raw_string_hashes)]
+
+use super::generate_docs;
+use crate::compilation::Compilation;
+
+#[test]
+fn callable_unit_types() {
+    let compilation = Compilation::new(
+        &[],
+        qsc::PackageType::Exe,
+        qsc::target::Profile::Unrestricted,
+    );
+    generate_docs(&compilation);
```

Comment:
Does this verify anything about the output? Or just that the call succeeded?

Reply:
This isn't a real unit test. This is the only way, currently, I have of triggering the generator. I'll need to have more clear direction on how we intend to use this and what to hook it up to before merging in.

Reply:
This has been replaced with an actual unit test now that we have the logic hooked up to npm.